attic/rfc_metrics.md

```diff
@@ -36,14 +42,24 @@ Internally, the function would look the following:
 
 Example:
 ```r
-fn = function(truth, response, positive, ...) {
-  assert_binary(truth, response = response, positive = positive)
-  fn_cm(cm(truth, response, positive))
+MeasureFairnessSimple = R6Class(
+inherits = "Measure",
+initialize = function(measure) {
+  self$measure = measure
+  # here do some other thing setting task types etc.
+  # minimize, properties, ...
+  self$minimize = measure$minimize
 }
+...
 
-fn_cm = function(m, na_value = NaN) {
-  m[2L, 1L]
-}
+$.score = function(task, operation, ...) {
```

Comment:
I think `operation` should not be passed during `$score` but during initialization. 
It should default to abs_diff

Reply:
I see, it should be set when creating the measure

Reply:
Emm, got one question. If we are gonna set the operation when initialize the fairness measures. Like
```
fair_measure = msr("groupwise_abs_diff", measure = base_measure)
```
Then there is no need to pass in the operation.

Reply:
I agree. 
Let us ignore the operation aspect for now and focus on getting absdiff to work.